auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue

```diff
@@ -194,12 +263,21 @@ export default class AccountInfo extends Mixins(AccountChangeMixin) {
   $refs: {
     editAccountForm: HTMLFormElement,
     mailingAddress:HTMLFormElement,
+    suspendAccountDialog:ModalDialog
   }
 
   private isFormValid (): boolean {
     return !!this.orgName || this.orgName === this.currentOrganization?.name
   }
 
+  private get isStaff (): boolean {
+    return this.currentUser.roles.includes(Role.Staff)
+  }
+
+  private isSuspendButtonVisible (): boolean {
+    return this.currentOrganization.statusCode === AccountStatus.ACTIVE || this.currentOrganization.statusCode === AccountStatus.SUSPENDED
```

Comment:
Hm..I think the logic is other wise ; If statusCode != NSF_SUSPENDED ; show the button ..we have other statuses as well..